airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/LabelInfo.tsx

```diff
@@ -26,6 +27,33 @@ const Description: React.FC<Pick<LabelInfoProps, "label" | "description">> = ({
   );
 };
 
+const Options: React.FC<Pick<LabelInfoProps, "options">> = ({ options }) => {
+  if (!options) {
+    return null;
+  }
+
+  return (
+    <div>
+      <h5 className={styles.optionsHeader}>
+        <FormattedMessage id="connector.optionsHeader" />
+      </h5>
+      <ul className={styles.options}>
+        {options.map((option) => (
```

Comment:
Could we add a bit of spacing in between these options? I think it would look a bit nicer if each option was slightly separated from the others